tools/Vrops.py

```diff
@@ -31,13 +31,13 @@ def get_token(target):
                                      timeout=10)
         except Exception as e:
             logger.error(f'Problem connecting to {target}. Error: {e}')
-            return False
+            return False, 522
 
         if response.status_code == 200:
-            return response.json()["token"]
+            return response.json()["token"], response.status_code
         else:
             logger.error(f'Problem getting token from {target} : {response.text}')
-            return False
+            return False, response.status_code
```

Comment:
What happens, if we don't have a status code?

Reply:
Will simply not happen. The `response` object is there at this point. Otherwise `response.text `never resolves.